### PR TITLE
Add max client connections as a parameter to the role

### PR DESCRIPTION
**Pull request by johnspicer**
@EDITD/hackers - This changes the zookeeper role to allow a max number of client connections to be set. By default it is 0 (unlimited)

### CODE REVIEW

**Comment by pkatseas** [comment]
LGTM :+1:

**Comment by johnspicer** [comment]
Also added a comment to explain. Thanks @pkatseas !

File: defaults/main.yml [hunk 1]
```diff
@@ -5,4 +5,6 @@ zookeeper_data_dir: /var/zookeeper
 zookeeper_data_log_dir: /var/zookeeper
 zookeeper_snap_retain_count: 3
 zookeeper_purge_interval: 2
+
+# 0 is unlimited - http://zookeeper.apache.org/doc/r3.4.5/zookeeperAdmin.html#sc_advancedConfiguration
```
Comment (pkatseas) [inline defaults/main.yml:9]: Good thinking!

**Comment by Fizzadar** [comment]
LGTM!

**Comment by johnspicer** [comment]
Thanks for the reviews @pkatseas @Fizzadar!

**Pull request merged**